src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/GATKSVVCFConstants.java

```diff
@@ -31,6 +32,8 @@
     public static final String INSERTED_SEQUENCE_MAPPINGS = "INSERTED_SEQUENCE_MAPPINGS";
     public static final String HOMOLOGY = "HOMOLOGY";
     public static final String HOMOLOGY_LENGTH = "HOMOLOGY_LENGTH";
+    public static final String ALT_HAPLOTYPE_SEQ = "ALT_HT_SEQ";
```

Comment:
Might as well spell out HAPLOTYPE here to make it clear in the VCF file

Reply:
done